docs/topics/db/models.txt

```diff
@@ -954,16 +954,42 @@ extend the parent's :ref:`Meta <meta-options>` class, it can subclass it. For ex
 
 Django does make one adjustment to the :ref:`Meta <meta-options>` class of an abstract base
 class: before installing the :ref:`Meta <meta-options>` attribute, it sets ``abstract=False``.
-This means that children of abstract base classes don't automatically become
-abstract classes themselves. Of course, you can make an abstract base class
-that inherits from another abstract base class. You just need to remember to
-explicitly set ``abstract=True`` each time.
```

Comment:
I don't see any changes in these lines except removing `Of course`. I will revert them. Please wrap docs at 79 chars in the future.

Reply:
I requested this change during review because "of course" we shouldn't use "of course" in docs; extracted to a full PR changing all docs in #12838.